cmd/gpu-kubelet-plugin/driver.go

```diff
@@ -39,11 +41,13 @@ import (
 const DriverPrepUprepFlockFileName = "pu.lock"
 
 type driver struct {
-	client       coreclientset.Interface
-	pluginhelper *kubeletplugin.Helper
-	state        *DeviceState
-	pulock       *flock.Flock
-	healthcheck  *healthcheck
+	client                  coreclientset.Interface
+	pluginhelper            *kubeletplugin.Helper
+	state                   *DeviceState
+	pulock                  *flock.Flock
+	healthcheck             *healthcheck
+	nvmlDeviceHealthMonitor *nvmlDeviceHealthMonitor
```

Comment:
Since we know that we will add new health monitors, let's rename the member to be more generic.
```suggestion
	deviceHealthMonitor *nvmlDeviceHealthMonitor
```